src/docker-utils.coffee

```diff
@@ -145,7 +145,7 @@ do ->
 							return true
 						return containerHasExited(containerInfo.Id)
 				.map (containerInfo) ->
-					docker.getContainer(containerInfo.Id).removeAsync()
+					docker.getContainer(containerInfo.Id).removeAsync(v: true, force: true)
```

Comment:
So adding the `v: true` here is huge, however one thing I wonder is how much this will help due to the `containerHasExited(containerInfo.Id)` check above (meaning it should be stopped and hence able to be removed anyway)